pkg/odo/cli/pipelines/add.go

```diff
@@ -0,0 +1,33 @@
+package pipelines
```

Comment:
This file collides with GITOPS-70.

Reply:
now it won't as we have made another environment directory